src/reconstruction/StandardModel.jl

```diff
@@ -204,19 +214,17 @@ end
 end
 
 @_remove_fn metabolite StandardModel String inplace begin
-    remove_metabolites!(model, [metabolite_id])
+    if !(metabolite_id in metabolites(model))
+        @_models_log @info "Metabolite $metabolite_id not found in model."
+    else
+        delete!(model.metabolites, metabolite_id)
+    end
+    nothing
```

Comment:
this breaks lots of stuff in a very ugly way. I'm perfectly okay with leaving the warning message there (that is actually very good!) but please do not produce inconsistent models. Details in slack. I'll make a patch with proper documentation later.

Reply:
Added the consistency check back in